aten/src/ATen/mps/MPSFallback.mm

```diff
@@ -54,8 +54,6 @@ Tensor slow_conv2d_forward_mps(
   m.impl("embedding_renorm_", torch::CppFunction::makeFromBoxedFunction<&mps_fallback>());
   m.impl("linalg_svd", torch::CppFunction::makeFromBoxedFunction<&mps_fallback>());
   m.impl("linalg_svd.U", torch::CppFunction::makeFromBoxedFunction<&mps_fallback>());
-  m.impl("_fft_c2c", torch::CppFunction::makeFromBoxedFunction<&mps_fallback>());
```

Comment:
Why remove ffts from here? This is not in the PR description.

Reply:
The enabling of FFT by default in Fallback is causing crashes in test_ops.py test and test Consistency tests.

Reply:
@albanD , can you please take a look?